decentraland/unity-renderer
# feat: show network and environment ribbon in development

## What does this PR change?

Fixes #993 
Also comes with a refactor of DebugController that is now a PluginFeature

## How to Test

- Run with `&NETWORK=mainnet` the ribbon should be OFF
- Run with `&NETWORK=ropsten` the ribbon should be ON
- If running with Unity, the ribbon is always ON

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
